task/t2.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from utils.read_files import read_files
+
+
+file_values = read_files()
+
+Temperatures = {}
+for file_value in file_values:
+    if file_value and file_value[1]:
+        Temperatures[file_value[0]] = float(file_value[1])
+
+dates_with_difference_of_7 = set()
+for date1, temp1 in Temperatures.items():
+    for date2, temp2 in Temperatures.items():
```

Comment:
you only need a single for loop

Reply:
As far as i understand the code, i understand that we cannot use a single loop to compare tow objects, but we use tow loops.